Sources/XcodeGenKit/ProjectGenerator.swift

```diff
@@ -41,15 +41,25 @@ public class ProjectGenerator {
         func getBuildEntry(_ buildTarget: Scheme.BuildTarget) -> XCScheme.BuildAction.Entry {
 
             let targetReference = pbxProject.objects.targets(named: buildTarget.target).first!
-            let target = project.getTarget(buildTarget.target)!
-            let buildableReference = XCScheme.BuildableReference(
-                referencedContainer: "container:\(project.name).xcodeproj",
-                blueprintIdentifier: targetReference.reference,
-                buildableName: target.filename,
-                blueprintName: buildTarget.target
-            )
-
-            return XCScheme.BuildAction.Entry(buildableReference: buildableReference, buildFor: buildTarget.buildTypes)
+            if let target = project.getTarget(buildTarget.target) {
```

Comment:
Fixes crash on explicitly unwrapped optional.

Reply:
Nice. Could we just get the `buildableName` in these `ifs`, and then keep the common code below those?

Reply:
Ah. I see. Copy paste code. Will fix it today later.